user_tools/src/spark_rapids_pytools/rapids/rapids_job.py

```diff
@@ -128,8 +130,19 @@ class RapidsLocalJob(RapidsJob):
     Implementation of a RAPIDS job that runs local on a local machine.
     """
 
-    def _build_classpath(self):
+    def _build_classpath(self) -> List[str]:
         deps_arr = [self.prop_container.get_jar_file()]
+        hadoop_confdir = Utilities.get_hadoop_conf_dir()
+        # append hadoop conf dir if any
+        if hadoop_confdir is not None:
+            try:
+                hadoopconf_path = LocalPath(hadoop_confdir)
+                # verify it is a valid directory
+                if hadoopconf_path.is_dir() and hadoopconf_path.exists():
+                    deps_arr.append(hadoopconf_path.no_prefix)
+            except Exception as e:  # pylint: disable=broad-except  # noqa: E722
+                self.logger.warning('Ignoring HADOOP_CLASSPATH. %s\n\tReason: Error while '
```

Comment:
what is going to throw an exception here.  Did you mean to say ignoring HADOOP_CLASSPATH or HADOOP_CONF_DIR?